ICSharpCode.Decompiler/IL/Transforms/TransformArrayInitializers.cs

```diff
@@ -143,6 +143,40 @@ internal static bool TransformSpanTArrayInitialization(NewObj inst, StatementTra
 			return false;
 		}
 
+		internal static bool TransformRuntimeHelpersCreateSpanInitialization(Call inst, StatementTransformContext context, out ILInstruction replacement)
```

Comment:
Thoughts on the method name?

Reply:
Not really, I am bad at naming things... 😅 ... luckily I had nine months to name my children...